reana_job_controller/htcondorcern_job_manager.py

```diff
@@ -129,7 +137,22 @@ def execute(self):
         job_ad["TransferInput"] = self._get_input_files()
         job_ad["TransferOutput"] = "."
         job_ad["PeriodicRelease"] = classad.ExprTree("(HoldReasonCode == 35)")
-        job_ad["MaxRunTime"] = 3600
+        if self.htcondor_max_runtime in (
+            "espresso",
+            "microcentury",
+            "longlunch",
+            "workday",
+            "tomorrow",
+            "testmatch",
+            "nextweek",
+        ):
```

Comment:
I think it would be great if we could extract this validation to R-Commons, and perform it in R-Client or R-Server (or even both).

Reply:
@diegodelemos could you elaborate? I agree it could be moved to r-commons however why/when should it be performed on r-client or r-server?

Reply:
Apologies for not being clear!

I had in mind the following scenario: people submitting e.g. `htcondor_max_runtime = epreso` (or any other typo) will see that their workflow runs but it will eventually specify no `JobFlavour` (it will use the default `MaxRunTime`) which is not what they meant.

If we move this check before in time and we make it more strict (e.g. if string check if it is in the list otherwise fail), we can be more correct and avoid unnecessary use of resources:
- If we do it in REANA-Client:
  - We avoid doing a call to the REANA cluster before the check passes
  - We give faster feedback to users (instead of waiting for minutes to realise that what they wanted didn't happen)
- If we do it in REANA-Server:
  - We avoid starting a new workflow with incorrect instructions (not the ones that the user wanted)
  - Centralising the check will also help in the case of allowing workflow creation from REANA-UI